src/vl-upload.js

```diff
@@ -147,8 +158,10 @@ export class VlUpload extends vlFormValidationElement(vlElement(HTMLElement)) {
             <button type="button" class="vl-upload__element__button vl-link">
               <i class="vl-vi vl-vi-paperclip" aria-hidden="true"></i>
               <span class="vl-upload__element__button__container"></span>
+              <slot id="title-slot" name="title"></slot>
```

Comment:
Kan je de slots niet beter rechtstreeks in de spans steken? Dan moet je zelf geen html liggen overkopieren?

Reply:
ja das wat de reden waarom ik nu het slot moet overzetten. Als ik deze bijv in de <small> tag stop wordt die overschreven door bijv de default i18n vertaling. En eens de component voorbij de fase van connectedCallback zit kan ik de vertaling ook niet meer aanpassen via de _changeTranslation methode. Initeel was dat wel het plan om het zo uit te werken.